test/unit/LivepeerETHTicketBroker.js

```diff
@@ -16,14 +15,16 @@ contract("LivepeerETHTicketBroker", accounts => {
     const sender = accounts[0]
     const recipient = accounts[1]
 
+    const unlockPeriod = 20
+
     before(async () => {
         fixture = new Fixture(web3)
         await fixture.deploy()
     })
 
     beforeEach(async () => {
         await fixture.setUp()
-        broker = await TicketBroker.new(fixture.controller.address, 0)
+        broker = await TicketBroker.new(fixture.controller.address, 0, unlockPeriod)
```

Comment:
The calls to Fixture in the `beforeEach` and `afterEach` hooks should reset the test chain state to whatever it was post `before` hook so we could move this line into the `before` hook to just deploy TicketBroker once

Reply:
👍